app/controllers/results_controller.rb

```diff
@@ -34,18 +34,14 @@ def results_params
   end
 
   def find_matching_bills
-    matching_bills = []
-    params[:matching_bills].each do |bill|
-      matching_bills << Bill.find_by(short_title: bill)
+    params[:matching_bills].map do |bill|
+      Bill.find(bill)
     end
```

Comment:
Nice! I really like this refactor. It's much more streamlined, cuts down on lines of code, and increases optimization.